src/node_http_parser_llhttp.cc

```diff
@@ -0,0 +1,17 @@
+#define NODE_EXPERIMENTAL_HTTP 1
```

Comment:
I was thinking about C++ CRTP but this looks like a cleverer approach 👍(hard to tell which one is harder to grok though XD)

Reply:
Yeah, this is … a bit unusual. But it’s the easiest approach when working from the current state, I think?